packages/core/integration-tests/test/scope-hoisting.js

```diff
@@ -4359,6 +4359,18 @@ describe('scope hoisting', function () {
       assert.deepEqual(output.default, obj);
     });
 
+    it('should add a default interop for a CJS module', async function () {
```

Comment:
This description could be made more specific, e.g.

```suggestion
    it('should add a default interop for a CJS module used dynamically in a hybrid module', async function () {
```